CN-global_conf.json

```diff
@@ -0,0 +1,204 @@
+{
+	"SX1301_conf": {
+		"lorawan_public": true,
+		"clksrc": 1,
+		"clksrc_desc": "radio_1 provides clock to concentrator",
+		"antenna_gain": 0,
+		"antenna_gain_desc": "antenna gain, in dBi",
+		"radio_0": {
+			"enable": true,
+			"type": "SX1255",
+			"freq": 486600000,
+			"rssi_offset": -176.0,
+			"tx_enable": true,
+			"tx_freq_min": 506700000,
+			"tx_freq_max": 508100000
+		},
+		"radio_1": {
+			"enable": true,
+			"type": "SX1255",
+			"freq": 487400000,
+			"rssi_offset": -176.0,
```

Comment:
@tftelkamp: this is `-166.0` in the other configs, is this change ok?

Reply:
This number should be provided by the gateway manufacturer. I don't know what a good default is for SX1255 based gateways.